testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketSslEchoTest.java

```diff
@@ -265,6 +267,7 @@ public void initChannel(Channel sch) throws Exception {
                 } else {
                     serverSslHandler = serverCtx.newHandler(sch.alloc());
                 }
+                serverSslHandler.setHandshakeTimeoutMillis(0);
```

Comment:
the motivation here is to rely upon the test timeout, makes debugging easier if there is a single timeout for the test but this isn't necessary.